clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -2264,6 +2264,25 @@ void CodeGenModule::EmitDeferred() {
   CurDeclsToEmit.swap(DeferredDeclsToEmit);
 
   for (GlobalDecl &D : CurDeclsToEmit) {
+    const ValueDecl *VD = cast<ValueDecl>(D.getDecl());
+    // If emitting for SYCL device, emit the deferred alias
+    // as well as what it aliases.
+    if (LangOpts.SYCLIsDevice && VD->hasAttr<AliasAttr>()) {
+      StringRef AliaseeName = VD->getAttr<AliasAttr>()->getAliasee();
```

Comment:
You can use one call of `getAttr` instead of doing `hasAttr` & `getAttr` both here as well.

Reply:
Done.